package.json

```diff
@@ -5,22 +5,27 @@
   "dependencies": {
     "@emotion/react": "^11.10.5",
     "@emotion/styled": "^11.10.5",
+    "@hookform/resolvers": "^2.9.10",
     "@mui/material": "^5.11.0",
     "@reduxjs/toolkit": "1.8.5",
     "axios": "0.27.2",
     "classnames": "^2.3.2",
     "formik": "^2.2.9",
+    "joi": "^17.7.0",
```

Comment:
"joi" using for validation of NODE JS API. Please use "yup" for the validation form. You can check how did it with your colleagues.

Reply:
I added yup